packages/bot-web-ui/src/components/route-prompt-dialog.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
+import { localize, Localize } from '@deriv/translations';
+import { Dialog } from '@deriv/components';
+import { connect } from '../stores/connect';
+
+const RoutePromptDialog = ({ continueRoute, is_confirmed, last_location, should_show, onCancel, onConfirm }) => {
+    useEffect(continueRoute, [is_confirmed, last_location]);
+
+    return (
+        <Dialog
+            title={localize('Are you sure?')}
+            confirm_button_text={localize('Yes')}
+            cancel_button_text={localize('No')}
+            onConfirm={onConfirm}
+            onCancel={onCancel}
+            is_visible={should_show}
+        >
+            <Localize i18n_default_text='Navgating to other platform will still complete current bought contract but will stop purchasing a new contract.' />
```

Comment:
Can you add a `TODO` comment here to update the text once it's available from copywriting? 
And please fix the typo for `Navigating` even if it's just a temporary placeholder.

Reply:
sure, thank you